packages/render-fidelity-tools/test/config.json

```diff
@@ -23,6 +19,10 @@
     {
       "name": "babylon",
       "description": "Babylon"
+    },    
+    {
+      "name": "gltfpt",
+      "description": "gltf-pt"
```

Comment:
Fidelity page align goldens according to the order of renderers here. i just want to put the gltfpt goldens to right most